app/src/views/OperationalLearnings/Filters/index.tsx

```diff
@@ -95,7 +109,20 @@ function Filters() {
                         placeholder={strings.filterOpsLearningsSearchPlaceholder}
                         value={rawFilter.search}
                         onChange={setFilterField}
+                        icons={<SearchLineIcon />}
                     />
+                    {showActions && (
+                        <div className={styles.actions}>
+                            <Button
+                                name={undefined}
+                                variant="secondary"
+                                onClick={resetFilter}
+                                disabled={!filtered}
+                            >
+                                {strings.opsLearningsFilterClear}
+                            </Button>
+                        </div>
```

Comment:
Let's not use `div` to wrap over the `Button`.  Also, we could pass this to the `actions` props of the `Container` component.